.circleci/config.yml

```diff
@@ -126,22 +126,17 @@ jobs:
       - image: "cimg/ruby:3.3.4-node"
     steps:
       - checkout
+      - ruby/install-deps
       - run:
-          name: Install ruby dependencies
+          name: Rubocop all files
+          when: always
           command: |
-            bundle install
-      - ruby/rubocop-check:
-          format: progress
-          label: Inspecting with Rubocop
```

Comment:
Idk how I feel about having Rubocop write on files at CI level. What the thought behind this rather than just having a check?

Reply:
This does just do a check:

Lint and fix
```bash
bundle exec rubocop --autocorrect
```

Lint only
```bash
bundle exec rubocop
```